azext_edge/edge/_help.py

```diff
@@ -1323,16 +1329,42 @@ def load_iotops_help():
         short-summary: Add a schema version to a schema.
         long-summary: |
                       To add a version, the associated storage account will need to have public network access enabled.
-                      For more information on the delta file format, please see
-                      https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-mapping and
-                      https://learn.microsoft.com/en-us/fabric/data-engineering/lakehouse-and-delta-tables. (#TODO AKA.ms)
+                      For more information on the delta file format, please see aka.ms/lakehouse-delta-sample
         examples:
         - name: Add a schema version 1 to a schema called 'myschema' within the registry 'myregistry' with
-                minimum inputs. The content is inline json.
+                minimum inputs. The content is inline json (powershell syntax example).
           text: >
-            az iot ops schema version add -n 1 -g myresourcegroup --registry myregistry --schema myschema --content '{\"hello\": 3}'
+            az iot ops schema version add -n 1 -g myresourcegroup --registry myregistry --schema myschema --content '{\\\"hello\\\": \\\"world\\\"}'
+        - name: Add a schema version 1 to a schema called 'myschema' within the registry 'myregistry' with
+                minimum inputs. The content is inline json (cmd syntax example).
+          text: >
+            az iot ops schema version add -n 1 -g myresourcegroup --registry myregistry --schema myschema --content "{\\\"hello\\\": \\\"world\\\"}"
+        - name: Add a schema version 1 to a schema called 'myschema' within the registry 'myregistry' with
+                minimum inputs. The content is inline json (bash syntax example).
+          text: >
+            az iot ops schema version add -n 1 -g myresourcegroup --registry myregistry --schema myschema --content '{"hello": "world"}'
```

Comment:
It seems a bit excessive to have examples for each various shell syntax - especially when powershell / cmd look exactly the same - can we just use bash but add a note that `inline json arguments should be escaped according to your shell's syntax` ?  We might have prior art from the previous extension on this.

Reply:
the prior art was examples for each terminal cause the people struggle... going to leave as is